backend/pkg/server/handler/extendapi.go

```diff
@@ -0,0 +1,45 @@
+package handler
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func New(kubeconfig string) (*ExtendApi, error) {
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		logrus.Errorf("kubeconfig error %s\n", kubeconfig)
+		return nil, err
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		logrus.Errorf("kubernetes clientset create error: %s", err.Error)
+		return nil, err
+	}
+
+	return &ExtendApi{config: config, clientset: clientset}, nil
```

Comment:
can we please init the kubeconfig client in the main method and then use it in the handler method.